python/requirements.txt

```diff
@@ -1,4 +1,5 @@
-construct==2.5.2
+construct==2.8.12
```

Comment:
setup.py reads this file into install_requires, so python will not allow sbp to be installed without this version of construct.